### DISCUSSION TITLE
I need help I've been hacked

### DISCUSSION THREAD

**Discussion by Bearroller9640** [community/community#139281] [New to GitHub]
I am not doing anything with the site hackers are using this site to hack me and upload child documents to hidden I. My phone all I'm trying 8 do is talk to s real person I support but this site flfoesy let you talk to s person I need help stop flagging my profile I. Not here to guide thts site just get help  please contact me asap  ive reported this to FCC and the government

**Comment by ghostinhershell** [comment]
Hi @Bearroller9640, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by Bearroller9640** [comment]
This is what my hacker is doing to my phone this didn't come from the site
I don't think but I'm begging for help police won't even look into to this
they think I'm making this up.  After 11 reports still no help FBI have 8
reports and don't help until police do. Can you suggest anyone to help me
I'm freaking scared to death.  I could end up in prison if there is hidden
child documents on my phone

**Comment by Bearroller9640** [comment]
I'm not into all this truck stuff I'm needing help he's doing damage and
police won't do anything
He's using this site to hack me
